modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm

```diff
@@ -30,6 +30,17 @@
 	icon_state = "hos_envirohelm_new"
 
 
+////////////////////////////
+//----- ACCESSORIES ------//
+////////////////////////////
```

Comment:
```
/*
*
*/
```
is a preferable style

Reply:
I dont know, but I'd have to change the whole file and Im too lazy for that lmao
This one is clean and more identifiable while you scroll IMO

Reply:
ok i just did them all